portals/publisher/src/main/webapp/source/src/app/components/Apis/Details/Configuration/components/WebSubConfiguration.jsx

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Rename this and other places with `WSO2 LLC` instead of `WSO2 Inc.`

Reply:
Fixed with https://github.com/wso2/apim-apps/compare/7ac7ff3e05faf284c2d238fab730ef6add2d4832..0bab035371a7d5842fb8a169b89bfcb1cbf6e6f0